### ISSUE TITLE
Decide where content team documentation should be placed

### ISSUE THREAD

**Issue by benoit74** [openzim/overview#23]
Currently, it looks like there are a lot of places where documentation about content edition is placed, and I probably miss some locations. 

This documentation is about:
- what is the overall process of content edition (high level picture of how do we go from a zim request to a published ZIM)
- how do we configure an offliner kind (youtube, mwoffliner, ...)

Some documentation is stored in various google docs (and I don't know where most of them are stored).

Some documentation is placed in the Zimfarm wiki (https://github.com/openzim/zimfarm/wiki), e.g. https://github.com/openzim/zimfarm/wiki/Ticket-Lifecycle-(Zimit), https://github.com/openzim/zimfarm/wiki/Tickets-Lifecycle-(Mwoffliner), https://github.com/openzim/zimfarm/wiki/Youtube-scraper-configuration-and-debug

And I just added a new location with [Kiwix content Google shared drive](https://drive.google.com/drive/u/1/folders/0AHJkIrd6O9UFUk9PVA).

I find this situation not convenient because:
- the various google docs are not centralized and will probably get lost at some point ; from my experience, every team document created in Google drive must be placed in a Google Shared drive and this Shared drive must be shared with appropriate persons of the team
- the documentation placed in Zimfarm wiki is mixed with very technical content for devs/ops
- Github wikis do not allow at all a review process, every change is immediately implemented in production
- Github wikis do not allow to have "folders" of documentation to structure stuff

From my perspective we should have:
- a central, public, reviewable, dedicated location for most documentation
- a very small Google shared drive (or something else) only for documentation which cannot be made public (like the API keys file I just created since these are secrets)

WDYT? Did I missed some locations? What would you suggest?

**Comment by benoit74** [comment]
Is this task considered challenging, making it unclear how to respond? Alternatively, is it possible that there is a lack of interest because documentation is perceived as a less prioritized topic?

**Comment by RavanJAltaie** [comment]
Hi Benoit, I agree with your points above.
We should have all documents placed in one place to be more accessible and organized.
What about storing everything in the shared drive you created earlier?
If agreed by @Popolechien , I'll start placing everything there.

**Comment by benoit74** [comment]
The problem with this Google Drive is that it is not a public repository (while most Kiwix documentation must be public AFAIK). I think however that we could split the drive in two parts, a public one (with most content) and a private one (with Youtube API keys for instance).

**Comment by Popolechien** [comment]
Not a huge fan of Google Drive for public-facing documents - I would limit its use to API keys and other sensitive info as you suggest.

@benoit74 I am not sure I understand your reluctance to use the Github wiki. As I see it at [/zimfarm/wiki](https://github.com/openzim/zimfarm/wiki) there is a clear table of content listing the different documents needed to run/operate the farm

> Github wikis do not allow at all a review process, every change is immediately implemented in production

This one sentence I did not understand. You mean there is no versioning (as opposite to a regular wiki)? Not ideal, but then if this is the only issue we could fall back to wiki.kiwix.org (not my preferred choice as I think staying close to the repo and limiting the number of platforms is key).

**Comment by benoit74** [comment]
@kelson42 then I suggest we discuss it live, this looks like a difficult discussion where we do not agree at all. It is very sad that your comment come so late in the discussion, it is not like if it couldn't have been anticipated

Who would like to participate (again) to the live discussion?

**Comment by Popolechien** [comment]
Yeah count me in. Though if I understand right that gothib actually means github there might not be too much of a disagreement.
